### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by nongdenchet** [beyonderVN/mytoto#1]
### Thank you for your submission
- You really pushed effort on doing the prework, I really appreciate that
### Good stuffs
- Using `RecyclerView` instead of `ListView` is great
- Using `ContentProvider` for data sharing
- You have tweak wonderful UI here, very very beautiful really appreciate that
- Nice to see the using of `DatePickerFragment` and `TimePickerFragment`
- Good implementation of `SQLite`
- Nice to see `ButterKnife` for readable code
- Your submission is the only one using `ModalBottomSheet`
- Using `newInstance` is a very good practice but I dont see you use it for your `DialogFragment`
### Suggestions
- Your features and UIs is awesome but the code is a little bit hard to follow. I do believe that we can work on this
### Questions
- How long have you developed Android? (just curious)
- What you expect learning from this course?

**Comment by beyonderVN** [comment]
Thank you so much for your prompt response

Here are my answers to your questions:
- I have developed Android for 5 month.
- I expect leaning new thing about Android development, working together
  on a team where we can share our experience and I want to make friend with
  other people have same passion.

**Comment by nongdenchet** [comment]
Nice to receive response from you. We will send accept email to you as soon as possible. Welcome on board and happy coding

**Comment by nongdenchet** [comment]
Hi have you receive confirmation email, please check your mail box

**Comment by beyonderVN** [comment]
i have received confirmation email, thank you for the information.
